cmd/cody-gateway/shared/config/config.go

```diff
@@ -297,7 +297,7 @@ func (c *Config) Load() {
 		c.AddError(errors.New("must provide allowed models for Google"))
 	}
 
-	c.AllowedEmbeddingsModels = splitMaybe(c.Get("CODY_GATEWAY_ALLOWED_EMBEDDINGS_MODELS", strings.Join([]string{string(embeddings.ModelNameOpenAIAda), string(embeddings.ModelNameSourcegraphSTMultiQA)}, ","), "The models allowed for embeddings generation."))
+	c.AllowedEmbeddingsModels = splitMaybe(c.Get("CODY_GATEWAY_ALLOWED_EMBEDDINGS_MODELS", strings.Join([]string{string(embeddings.ModelNameOpenAIAda), string(embeddings.ModelNameSourcegraphSTMultiQA), string(embeddings.ModelNameSourcegraphMetadataGen)}, ","), "The models allowed for embeddings generation."))
```

Comment:
For readability, consider teasing the model names out into a separate variable. (To just avoid the horizontal scrolling.

```suggestion
	defaultEmbeddingModles := []string{
		string(embeddings.ModelNameOpenAIAda),
		string(embeddings.ModelNameSourcegraphSTMultiQA),
		string(embeddings.ModelNameSourcegraphMetadataGen)}
	configuredEmbeddingModels := c.Get(
		"CODY_GATEWAY_ALLOWED_EMBEDDINGS_MODELS",
		strings.Join(allowedEmbeddingsModels, ","),
		"The models allowed for embeddings generation.")
	c.AllowedEmbeddingsModels = splitMaybe(configuredEmbeddingModels)
```